admin_guide/allocating_node_resources.adoc

```diff
@@ -101,6 +101,50 @@ status:
 ----
 ====
 
+[[system-resources-reported-by-node]]
+== System resources reported by node
+
+Since OpenShift 3.3 each node reports system resources utilized by the container runtime and kubelet.
+To better aid your ability to configure *--system-reserved* and *--kube-reserved*,
+you can introspect the nodes resource usage using the node summary api which is accessible at *<master>/api/v1/nodes/<node>/proxy/stats/summary*.
+
+For instance, to access the resources from *cluster.node22* node, you can run:
+
+----
+$ curl <certificate details> https://<master>/api/v1/nodes/cluster.node22/proxy/stats/summary
```

Comment:
We can direct them to the REST API Overview topic here:

https://docs.openshift.com/enterprise/latest/rest_api/index.html

If we need to add any more general info on certs/auth, we can add them there.